umbraco/Umbraco-CMS
# Netcore: Fixes issues with user invites

### Details
Fixes issues with user invites
- Issue with the generated link in the invite email. The querystring was not added
- Allow anonymous access to CurrentUserController.PostSetInvitedUserPassword, as it is used by users not logged in
- Allow anonymous access to AuthenticationController.GetPasswordConfig, as this is used to set a password for newly invited users, before they login

### Tests
- Add SMTP info in config (I use [smtp4dev](https://github.com/rnwood/smtp4dev) to create a local fake smtp server)
- Invite user from user section
- Follow the link in the mail
- Ensure the user can create password and login
